packages/@webex/plugin-meetings/src/media/properties.ts

```diff
@@ -259,10 +259,13 @@ export default class MediaProperties {
     const allStatsReports = [];
 
     try {
-      // eslint-disable-next-line no-await-in-loop
       const statsResult = await this.webrtcMediaConnection.getStats();
 
-      statsResult.forEach((report) => allStatsReports.push(report));
+      if (typeof statsResult.forEach === 'function') {
```

Comment:
This is because depending on the type of media connection we get the stats reports in different formats, either a js object or a RTCStatsReport object.  This chunk of code is to unify them.

Reply:
I think I would prefer to unify them at WCME or internal-media-core level. Longer term we want the RoapMediaConnection and MultistreamRoapMediaConnection classes to have the same API interface. We're calling getStats() here and I think it's reasonable to expect that the result has a forEach() method (like the normal webrtc getStats() result does), so can we change WCME to return an object that has forEach() from getStats()?

Reply:
I talked to the WCME team and in WCME they break down and inject csi and mid into the stats, that's why it is a js object instead of a `RTCStatsReport`.  Since we need csi in the MQE but don't have it yet, should we try to do the same thing for RoapMediaConnection in internal-media-core?